src/Command/MigrateGestionnaireToMemberCommand.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\ClientMembre;
+use App\Entity\Gestionnaire;
+use App\Entity\Membre;
+use App\Entity\MembreCentre;
+use App\Entity\User;
+use App\Repository\CentreRepository;
+use App\Repository\ClientGestionnaireRepository;
+use App\Repository\GestionnaireRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'app:migrate-gestionnaire-to-member',
+    description: 'Migrate gestionnaire user to new member user',
+)]
+class MigrateGestionnaireToMemberCommand extends Command
+{
+    public function __construct(
+        private readonly GestionnaireRepository $gestionnaireRepository,
+        private readonly EntityManagerInterface $em,
+        private readonly CentreRepository $relayRepository,
+        private readonly ClientGestionnaireRepository $clientGestionnaireRepository,
+        string $name = null,
+    ) {
+        parent::__construct($name);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $gestionnaires = $this->gestionnaireRepository->findAll();
+        $newMembersCount = 0;
+        $removedGestionnaires = 0;
+
+        $io->info(sprintf('Found %d gestionnaires', count($gestionnaires)));
+
+        $progressBar = new ProgressBar($output, count($gestionnaires));
+        $progressBar->start();
+
+        foreach ($gestionnaires as $gestionnaire) {
```

Comment:
You can use the progress bar this way so you don't have to start() and finish() :
```
$progressBar = new ProgressBar($io);
foreach ($progressBar->iterate($gestionnaires) as $gestionnaire) {
```